README.md

```diff
@@ -101,6 +101,31 @@ You can then build as usual with CMake:
 cmake --build build-x86_64-hermit-debug
 ```
 
+## Autoconf
+
+When using Autoconf, the build directory can be set up for Hermit:
+
+First create and enter the build directory:
+
+```bash
+mkdir build-hermit
+cd build-hermit
+```
```

Comment:
I like this, (and I was not aware that this works in autotools). But shouldn't we expain the simple case instead and explain the build directory second/link to some documentation?

Reply:
I think having a build directory is good practice and helps organize the project by separating source from build artifacts. I have seen many build scripts do this. Personally, I don't think there is much overhead to this, and this may even help propagate the good practice. :D